lang/rs/bindings/cners/src/port.rs

```diff
@@ -0,0 +1,107 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2020-2022 Intel Corporation.
+ */
+
+use super::bindings::bindings::{
+    _pktdev_rx_burst, _pktdev_tx_burst, lport_stats_t, pktdev_buf_alloc, pktdev_stats_get,
+};
+use super::error::*;
+use super::packet::*;
+use std::mem::MaybeUninit;
+
+pub struct Port {
```

Comment:
This is similar to the Go code and we do not need a structure for just a port number. This needs to be fixed in the Go code too.

Reply:
I used Port is an abstract data type used to represent an CNE lport. This has both data and functions just like a C++ class which allows to send/recv packets from an lport.